src/paymentservice/charge.js

```diff
@@ -9,6 +9,7 @@ const { FlagdProvider} = require('@openfeature/flagd-provider');
 const flagProvider = new FlagdProvider();
 
 const logger = require('./logger');
+const runtimeMetrics = require('./runtimeMetrics');
```

Comment:
Are we instrumented the nodejs app using two approaches at the same time, the one from community vs. the one from our side? Instead of modifying the business code, can we move this to opentelemetry.js?

Reply:
Yes, but we can't move it because it is in the manual instrumention process.

Reply:
I have moved the metrics collection function to opentelemetry.js.